### PR TITLE
fixed the displayed sections

### PR DESCRIPTION
**Pull request by hodzy**

### CODE REVIEW

**Comment by hodzy** [comment]
comment on the issue:
     1- isTutor is set to false when tutor is displayed on the side sreen

File: src/routes/Settings.tsx [hunk 1]
```diff
@@ -99,12 +98,15 @@ const Settings: React.FC = () => {
...
   };
-
   return (
     <div className={classes.container}>
       <Title>Deine Informationen</Title>
       <SettingsCard user={userContext.user} />
-      {!isProjectCoachButNotTutor(userContext.user) && renderSubjects()}
+      {!userContext.user.isTutor &&
+        !(
+          userContext.user.isProjectCoach || userContext.user.isProjectCoachee
```
Comment (lsc0498) [inline src/routes/Settings.tsx:107]: Why do you exclude project coaches here?

File: src/routes/Settings.tsx [hunk 2]
```diff
@@ -99,12 +98,15 @@ const Settings: React.FC = () => {
       </>
     );
   };
-
   return (
     <div className={classes.container}>
       <Title>Deine Informationen</Title>
       <SettingsCard user={userContext.user} />
-      {!isProjectCoachButNotTutor(userContext.user) && renderSubjects()}
+      {!userContext.user.isTutor &&
```
Comment (lsc0498) [inline src/routes/Settings.tsx:105]: If the user is tutor, he SHOULD see the subjects.

Comment (hodzy) [reply src/routes/Settings.tsx:105]: @lsc0498 i tried to explain it in a comment above
but here it is again

-> isTutor is set to false when tutor is displayed on the side sreen in thapp(so for now we wrote the code to function with what we have
on the screen)
bec. i don't know exactly if the problem is only in the frontend or is there also a problem in the backend

File: src/routes/Settings.tsx (hunk 1 above)
Comment (hodzy) [reply src/routes/Settings.tsx:107]: @lsc0498 if i remember correctly, only tutors get to see the "Deine Fächer" part and only project coaches get to see "Deine Projekte"

Comment (lsc0498) [reply src/routes/Settings.tsx:107]: Yes that's right. But a project coach can also be a tutor and in this case he should see both.

File: src/routes/Settings.tsx (hunk 2 above)
Comment (lsc0498) [reply src/routes/Settings.tsx:105]: What do you mean by side screen?

File: src/routes/Settings.tsx (hunk 1 above)
Comment (hodzy) [reply src/routes/Settings.tsx:107]: will change it as soon as i can then :)

File: src/routes/Settings.tsx (hunk 2 above)
Comment (hodzy) [reply src/routes/Settings.tsx:105]: in the part where you see the settings and 1 on 1 mentoring... meant the side menu

Comment (hodzy) [reply src/routes/Settings.tsx:105]: Capture
Capture2

Comment (lsc0498) [reply src/routes/Settings.tsx:105]: There it should only be indicated whether the user is "Student*in" or "Schüler*in" which actually should have nothing to do with `isTutor`.

Comment (hodzy) [reply src/routes/Settings.tsx:105]: @lsc0498 examples of the problem above.. here i console.log the value for the isTutor boolean variable, and here ou can see that when isTutor is true -> it is not shown as a tag and vice versa

Comment (lsc0498) [reply src/routes/Settings.tsx:105]: I think the problem is that you haven't pulled before making this branch ;) Please merge `master` into your branch.

Comment (hodzy) [reply src/routes/Settings.tsx]: ooh :) , thx .. will try to fiy the rest as soon as possible

**Pull request closed without merging**